hub/core/tests/test_storage_provider.py

```diff
@@ -2,25 +2,28 @@
 from hub.core.storage.mapped_provider import MappedProvider
 import pytest
 from hub.util.check_s3_creds import s3_creds_exist
+from hub.util.cache_chain import get_cache_chain
 
+MB = 1024 * 1024
 
-def test_provider(provider=MappedProvider()):
+
+def test_storage_provider(provider=MappedProvider()):
```

Comment:
Pytest is going to run this as its own test if it has the `test_` prefix. Maybe change it to `check_` so it's clear that it's a helper

Reply:
We need to test MappedProvider too. So running this test is fine